Marlin/src/pins/stm32f4/pins_LERDGE_S.h

```diff
@@ -150,64 +148,83 @@
 //
 // Misc. Functions
 //
-#define SDSS                                PC11  // SD is working using SDIO, not sure if this definition is needed?
 #define LED_PIN                             PC6   // Mainboard soldered green LED
 #define PS_ON_PIN                           PB2   // Board has a power module connector
-#define KILL_PIN                            -1    // There is no reset button on the LCD
-#define POWER_LOSS_PIN                      -1    // PB2 could be used for this as well
+#define KILL_PIN                              -1  // There is no reset button on the LCD
+#define POWER_LOSS_PIN                        -1  // PB2 could be used for this as well
 
 //
 // SD support
 //
 #define SDIO_SUPPORT
 #define SDIO_CLOCK                       4800000
-
-#define SD_SCK_PIN                          PC12
-#define SD_MISO_PIN                         PC8
-#define SD_MOSI_PIN                         PD2
-#define SD_SS_PIN                           PC11
-
 #define SD_DETECT_PIN                       PG15
+#if DISABLED(SDIO_SUPPORT)
+  #define SOFTWARE_SPI
+  #define SD_SCK_PIN                        PC12
+  #define SD_MISO_PIN                       PC8
+  #define SD_MOSI_PIN                       PD2
+  #define SD_SS_PIN                         PC11
+  #define SDSS                              PC11
+#endif
 
 //
 // Persistent Storage
 // If no option is selected below the SD Card will be used
-// (this section modelled after pins_LONGER3D_LK.h)
-// Warning: Not tested yet! Pins traced with multimeter, mistakes are possible
+// Prefer the I2C one (F-RAM) to store marlin settings, SPI one is not working yet
+
 //#define SPI_EEPROM
+//#define I2C_EEPROM
 
 #if ENABLED(SPI_EEPROM)
-  // Lerdge has an SPI EEPROM Winbond W25Q128 (128Mbits) https://www.pjrc.com/teensy/W25Q128FV.pdf
+  // SPI EEPROM Winbond W25Q128 (128Mbits) https://www.pjrc.com/teensy/W25Q128FV.pdf
   #define SPI_CHAN_EEPROM1                     1
   #define SPI_EEPROM1_CS_PIN                PB12  // datasheet: /CS pin, found with multimeter, not tested
   #define EEPROM_SCK_PIN                    PB13  // datasheet: CLK pin, found with multimeter, not tested
   #define EEPROM_MISO_PIN                   PB14  // datasheet: DO pin, found with multimeter, not tested
   #define EEPROM_MOSI_PIN                   PB15  // datasheet: DI pin, found with multimeter, not tested
   #define EEPROM_PAGE_SIZE               0x1000U  // 4KB (from datasheet)
   #define MARLIN_EEPROM_SIZE 16UL * (EEPROM_PAGE_SIZE)   // Limit to 64KB for now...
+#elif ENABLED(I2C_EEPROM)
+  // FM24CL64BG (CYP1813) 64Kbit F-RAM
+  #define SOFT_I2C_EEPROM                         // Force the use of Software I2C
+  #define I2C_SDA_PIN                       PG13
+  #define I2C_SCL_PIN                       PG14  // To be confirmed on the Lerdge S, but probably same as the K
```

Comment:
SDA/SCL to pins to be confirmed... those seems Y/ZMAX for now

Reply:
This picture show the F-RAM, upside down vs the K... so the important pins to find are to 2 on the top right

![image](https://user-images.githubusercontent.com/145119/131330493-093347ef-2d06-4305-8e56-4c65dfe352de.png)